### PR TITLE
TodoList

### PR DESCRIPTION
**Pull request by itsAakashz**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by Priyansxu** [comment]
William bhaiya, here's your promised girlfriend 📈📈📈

**Comment by itsAakashz** [comment]
plz

**Comment by Priyansxu** [comment]
Go and sleep now

**Comment by itsAakashz** [comment]
first merge it na

**Comment by orxngc** [comment]
Oh and I just realized the todolist.json is in the root folder, not the domain folder. kek

**Review by triach-rold — APPROVED** [review]
[This](itsAakashz.github.io) website is reachable and complete.

**Review by triach-rold — CHANGES_REQUESTED** [review]
[This](todolist-ec6.pages.dev) is not reachable. Also, this is not in the domains folder.

**Comment by itsAakashz** [comment]
Can i deploy this website anywhere apart from cloudflare like Vercel, Render, Netlify etc?

**Comment by CuteDog5695** [comment]
> Can i deploy this website anywhere apart from cloudflare like Vercel, Render, Netlify etc?

render is fine, vercen and netlify no

File: domains/itsAakashz
Comment (CuteDog5695) [inline domains/itsAakashz:1]: file isnt json

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/itsAakashz
Comment (itsAakashz) [reply domains/itsAakashz:1]: now check website is live.. now plz merge it.

File: todoList.json
Comment (CuteDog5695) [inline todoList.json:1]: this file isnt in the domain directory

File: todoList.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+{
+  "owner": {
+    "username": "itsAakashz",
+    "email": "techtutez@gmail.com",
+    "twitter": "itsAakashz"
+  },
+  "record": {
+    "CNAME": "todolist-ec6.pages.dev"
```
Comment (CuteDog5695) [inline todoList.json:9]: site isnt accessible

**Review by iostpa — CHANGES_REQUESTED** [review]

File: todoList.json (hunk 1 above)
Comment (itsAakashz) [reply todoList.json:9]: i don't want this site.. check last commit

Comment (CuteDog5695) [reply todoList.json:9]: then delete the file mate

Comment (itsAakashz) [reply todoList.json:9]: oh wait

**Comment by itsAakashz** [comment]
Check my Aakash.json

**Pull request closed without merging**